docs/export-to-csv.md

```diff
@@ -0,0 +1,59 @@
+
+![Export To CSV Action Configuration](images/export-to-csv-action-configuration.png)
+
+Select `Apply` when you are ready to move forward to the next step of using this Action.
+
+## Executing the Action and Downloading the CSV
+
+After selecting your Export ADOs to CSV Configuration Options, you will be directed to a page showing a summary list of the 'Items selected' (list may be truncated depending on the amount of items you selected). You will then need to select `Execute Action` to proceed.
+
+Selecting `Execute Action` will trigger the generation of the CSV in real time, directly in your browser session. This may take a few seconds to a few minutes, depending on the amount of items you selected to include. You will see a progress bar updating while this Action is running.
+
+When the Action is finished, you will see a temporary Status Message stating: "Your source data was saved and is available as CSV at", followed by a link to the CSV file generated. Be sure to click on the CSV link and download before navigating away from this temporary Status message. No worries if you accidently navigate away from this page before downloading the generated CSV, you can find a temporary copy of the CSV file generated at:
+- Accessed through the `Content` menu > `Media Files`
+- Directly at `/admin/content/files`
+
+*Or:* you can simply configure and run the Action again to generate another copy of the CSV if needed.
+
+If you selected the option to attach to `Attach CSV to a new AMI Set`, you will also see a status message stating: "Well Done! New AMI Set was created and you can see it here" (and 'see it here' will be linked to the corresponding AMI Set). You will be able to access the AMI Set from the `AMI Sets` tab on the main Content page found at `/admin/content` or directly at `/amiset/list`.
+
+![Export To CSV Action Status Message](images/export-to-csv-status-message.png)
+
+## Recommendations
+
+This specialty Action is quite useful for generating CSVs to use for reviewing select batches of materials found in your repository, but it is not intended to be used to generate a complete CSV containing all of your Archipelago repository's digital objects and collections. Depending on the size of your repository, generating a CSV for all of your assets may exceed your browser's timeout processing limits/bandwidth and also result in a CSV file that is unable to be opened by standard spreadsheet software such as Google Sheets or MS Excel.
+
+We would recommend executing this specialty Action using Archipelago's [Find and Replace](find_and_replace.md) to narrow down your selected ADOs more easily using the Facets available for the Find and Replace interface. We would also recommend keeping your selected batches limited to an amount of objects you can reasonably assess and review using your normal spreadsheet review workflows and software.
```

Comment:
@alliomeria maybe this is a good doc to point to the fact that Solr has a fixed limit, so if you make a LARGE query and select all and your fixed limit is lower, you will get less records? Like a note as `Known ISSUES` 
at /admin/config/search/search-api/server/esmero_solr/edit
![image](https://github.com/user-attachments/assets/079a2c42-7c1a-404e-be58-36e09a6651cc)

Reply:
Ok, I can add info expanding upon this more specifically. Will circle back to this and update my local fork pushing this.

Reply:
Danke!